backend/types/Game.ts

```diff
@@ -0,0 +1,14 @@
+export class Game {
```

Comment:
you don't instantiate either --> either use it, or simply turn it into an interface

Reply:
Fixed in https://github.com/salsita/foosball-rating/pull/113/commits/f657e38aac9173ce08efb3ec3cf4c740fc7b689b